Odata-docs/webapi/query-options-in-request-body.md

```diff
@@ -0,0 +1,118 @@
+---
+title:  "Passing OData query options in request body"
+date:   2020-09-28 07:49:00
+
+ms.date: 9/28/2020
+---
+# Passing OData query options in request body
+**Applies To**:[!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-core-v7.5.md)][!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v7.5.md)]
+
+The query options part of an OData URL can be quite long, potentially exceeding the maximum length that many hosting environments (including IIS) impose. This can limit the client's ability to request the exact set of data they are interested in, especially when using explict select statements.
+
+OData AspNet WebApi V7.5 introduced support for passing query options in the request body. This is achieved by prepating a POST request with the query options part of the URL in the body and sending that to an endpoint comprising of the resource path appended with `/$query` - e.g. http://ServiceRoot/Movies/$query. The `Content-Type` header should be set to `text/plain`. 
```

Comment:
prepating? can it be a simple/common word.

Reply:
The word should be "preparing". I'll fix and push the change